orientdb/src/main/scala/akka/stream/alpakka/orientdb/OrientDBSinkSettings.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.orientdb
+
+import com.orientechnologies.orient.core.db.OPartitionedDatabasePool
+
+//#sink-settings
+final case class OrientDBSinkSettings(oDatabasePool: OPartitionedDatabasePool,
+                                      maxPartitionSize: Int = Runtime.getRuntime.availableProcessors(),
+                                      maxPoolSize: Int = -1,
+                                      maxRetry: Int = 1,
+                                      retryInterval: Int = 5000,
```

Comment:
Can you make `retryInterval` a `Duration`, please. ...and adopt the Java `withRetries` to take Long and TimeUnit.